docker-compose.yml

```diff
@@ -1,32 +1,39 @@
 version: "3.5"
 services:
-  user_app:
+  cdslab_cmodels_app:
     build: .
-    container_name: cdslab_models_app
+    container_name: cdslab_cmodels_app
     env_file: .env
     image: cdslab_user
     networks:
-      - cdslab_models
+      - cdslab_cmodels_net
+    depends_on:
+      - cdslab_cmodels_mongo
     ports:
       - 5000:5000
     volumes:
-    - ./src:/app/src
+    - cdslab_cmodels_app_data:/app/src
 
-  user_mongo:
-    container_name: cdslab_models_mongo
+  cdslab_cmodels_mongo:
+    container_name: cdslab_cmodels_mongo
     environment:
       MONGO_INITDB_ROOT_USERNAME: cdsuser
       MONGO_INITDB_ROOT_PASSWORD: cdspass
     image: mongo:3-xenial
     networks:
-      - cdslab_models
+      - cdslab_cmodels_net
     ports:
       - 27017:27017
     volumes:
-      - /tmp/data/cdslab_models/:/data/db
-
+      - cdslab_cmodels_mongo_data:/tmp/data/cdslab_cmodels_mongo/
 
 networks:
-  cdslab_models:
-    name: cdslab_user
-    driver: bridge
\ No newline at end of file
+  cdslab_cmodels_net:
+    name: cdslab_cmodels_net
+    driver: bridge
+
+volumes:
+  cdslab_cmodels_app_data:
+    driver: local
+  cdslab_cmodels_mongo_data:
+    driver: local
```

Comment:
There is not necessary, because by default, all drivers are _local_ , and in previous syntax, use default volume in short syntax:
i.e.
```
...
volumes:
    - /tmp/data:/data/db
 ...
 ```
 `- ${LOCAL_PATH}:${CONTAINER_PATH}`

Reply:
Ok I will delete this section